modules/perception/obstacle/camera/lane_post_process/common/type.h

```diff
@@ -34,6 +34,10 @@
 namespace apollo {
 namespace perception {
 
+#ifndef MAX_LANE_HISTORY
```

Comment:
I would suggest using const rather than macro. See https://google.github.io/styleguide/cppguide.html#Local_Variables (Instead of using a macro to store a constant, use a const variable. Instead of using a macro to "abbreviate" a long variable name, use a reference.)

Reply:
thanks! will change it to const  soon.